src/main/java/com/solopov/urlshorter/exceptions/LinkNotFoundException.java

```diff
@@ -0,0 +1,4 @@
+package com.solopov.urlshorter.exceptions;
+
+public class LinkNotFoundException extends RuntimeException {
```

Comment:
use consistent naming ... Url is already used everywhere else

Reply:
Changed